README.md

```diff
@@ -14,9 +14,9 @@ This repository contains the example code material for the SC24 tutorial:
 
 ## Links
 
-Tutorial slides: https://drive.google.com/drive/folders/1wN1bCjHk2iocI6nowuzSugQopAwetCjR?usp=sharing
+Tutorial slides: https://drive.google.com/drive/u/2/folders/1IfHYBBduBOobWEHzeuzoL8zGSKyTGKj9
```

Comment:
you should use the URLs returned by the share option in google drive. You can also just remove the `u/2/` part because that means it will open in the user's 3rd google account.